addons/project/project.py

```diff
@@ -440,6 +440,15 @@ def write(self, cr, uid, ids, vals, context=None):
             vals.update(alias_model_id=model_ids[0])
         return super(project, self).write(cr, uid, ids, vals, context=context)
 
+    def _find_create_partner_from_mail(self, cr, uid, email_list, name_email, context=None):
```

Comment:
can you move this method in res.partner model in base module ?

and also check this method: find_or_create in res.partner. can not use this method ?